kzg_prover/src/circuits/tests.rs

```diff
@@ -85,26 +85,26 @@ mod test {
             full_prover(&params, &pk, circuit.clone(), vec![vec![]]);
```

Comment:
Specify what is this ZK proof and how it differs from the other proofs that we are generating later